docs/guides/mobile-plugin-ima-android.md

```diff
@@ -67,3 +68,11 @@ To install with Maven, add the SDK as a dependency in the `pom.xml` file:
   <version>0.5.0</version>
 </dependency>
 ```
+
+## Notes for using R8 or ProGuard
+
+If you are using R8 the shrinking and obfuscation rules are included automatically.
```

Comment:
If you are using R8, the shrinking and obfuscation rules are included automatically.

Reply:
fixed